content/en/apps/guides/forms/rdtoolkit-integration.md

```diff
@@ -0,0 +1,201 @@
+    * The available options can be found in the [RDToolkit app source code](https://github.com/dimagi/rd-toolkit).
+    * Assign a group name to enable all the RDT associated with the group, example: `mal_pf`.
+    * Assign individual RDT names to limit the options available for the user, separated by one space, example: “sd_bioline_mal_pf_pv carestart_mal_pf_pv”.
+
+Optional fields:
+* `rdtoolkit_api_url` | Text
+     * API URL that will receive the RDT results, sent by RDToolkit app via `POST`. See [RDToolkit app source code](https://github.com/dimagi/rd-toolkit) for more details.
+* `rdtoolkit_state` | Text
+     * The RDT status.
+     * Auto filled by the CHT integration.
+* `rdtoolkit_time_started` | Text
+     * The date and time when the RDT started.
+     * Auto filled by the CHT integration.
+* `rdtoolkit_time_resolved` | Text
+     * The date and time when the RDT result is ready to capture.
+     * Auto filled by the CHT integration.
+
+### Sample form for provisioning RDT
+![Provisioning RDT form](rdtoolkit-povision-form.png)
+
+### Capturing RDT result form
+It is required to add the following labels in the “appearance” tag:
+* `rdtoolkit-capture`: To attach the Enketo widget that connects with the RDToolkit app.
+* `rdtoolkit-action-btn`: To attach the action button that captures the RDT result, this button will open the RDToolkit.
+
+This form has the following mandatory fields:
+* `patient_uuid` | Text
+    * Patient's unique ID in the system.
+    * Auto filled when using the contact lookup field.
+* `patient_id` | Text
+    * Patient's representative ID, for example, can be the ID in the health facility's records.
+    * Auto filled when using the contact lookup field.
+* `patient_name` | Text
+    * Patient's first name and / or last name
+    * Auto filled when using the contact lookup field.
+* `rdtoolkit_session_id` | Text
+    * The unique ID for the RDT.
+    * Auto filled by CHT integration.
+* `rdtoolkit_results` | Text
+    * JSON representation of the RDT result
+    * Auto filled by CHT integration.
+* `rdtoolkit_results_description` | Text
+    * Translation of the results
+    * Auto filled by CHT integration.
+* `rdtoolkit_image` | Text
+    * Base64 image encoding.
+    * Auto filled by CHT integration and will render the image.
+    * **appearance**: hidden
+    * **instance::type**: binary
+
+Optional fields:
+* `rdtoolkit_state` | Text
+    * The RDT status.
+    * Auto filled by CHT integration.
+* `rdtoolkit_time_started` | Text
+    * The date and time when RDT started.
+    * Auto filled by CHT integration.
+* `rdtoolkit_time_resolved` | Text
+    * The date and time when the RDT result is ready to capture.
+    * Auto filled by CHT integration.
+* `rdtoolkit_time_read` | Text
+    * The date and time when RDT result was captured.
+    * Auto filled by CHT integration.
+
+### Sample form for capturing RDT results
+![Capturing RDT result form](rdtoolkit-capture-form.png)
```

Comment:
Same question about this image and if it works as expected.

Reply:
It turns out that the folder needs to be called the same as this file and it doesn't need path
Fixed!